.azure-pipelines/MacOS-CI.yml

```diff
@@ -10,13 +10,34 @@ jobs:
       py27:
         python.version: '2.7'
         onnx_ml: 0
+        onnx_debug: 0
+        onnx_lite: OFF
+      py27-ml:
+        python.version: '2.7'
+        onnx_ml: 1
+        onnx_debug: 0
+        onnx_lite: OFF
+      py27-ml-lite:
+        python.version: '2.7'
+        onnx_ml: 1
+        onnx_debug: 0
+        onnx_lite: ON
       py36:
         python.version: '3.6'
         onnx_ml: 0
-      py36-onnx-ml:
+        onnx_debug: 0
+        onnx_lite: OFF
+      py36-ml:
         python.version: '3.6'
         onnx_ml: 1
-    maxParallel: 3
+        onnx_debug: 0
+        onnx_lite: OFF
+      py36-ml-debug:
+        python.version: '3.6'
+        onnx_ml: 1
+        onnx_debug: 1
+        onnx_lite: OFF
```

Comment:
${onnx_lite} is empty. I think you need to put OFF or ON inside quotation marks

This is from MACOS CI for this PR
ONNX version          : 1.7.0
--   ONNX NAMESPACE        : ONNX_NAMESPACE_FOO_BAR_FOR_CI
--   ONNX_BUILD_TESTS      : ON
--   ONNX_BUILD_BENCHMARKS : OFF
--   ONNX_USE_LITE_PROTO   : 
--   ONNXIFI_DUMMY_BACKEND : OFF
--   ONNXIFI_ENABLE_EXT    : OFF